impeller/playground/imgui/imgui_impl_impeller.cc

```diff
@@ -78,8 +79,8 @@ bool ImGui_ImplImpeller_Init(
     int width, height;
     io.Fonts->GetTexDataAsRGBA32(&pixels, &width, &height);
 
-    auto texture_descriptor = impeller::TextureDescriptor{};
-    texture_descriptor.storage_mode = impeller::StorageMode::kHostVisible;
+    impeller::TextureDescriptor texture_descriptor;
```

Comment:
Not really related but this was also tripping validation errors for me, because molten vk actually creates the linear texture but then we can't actually do anything with it.